README.md

```diff
@@ -57,6 +57,15 @@ If you want to **add a new component**:
 
 If the answer to any of these questions is no, chat with your teammates about how best to achieve what you want. Don’t be afraid to ask questions in the `#product-prism-support` Slack channel.
 
+### File Structure
+Each component should have a few files associated with it:
+- The component itself (`.jsx`)
+- Any necessary styling (`.scss`)
+- Component documentation (`.docs.mdx`)
+- Component stories (`.story.jsx`)
+
+The component and styles work together as you would expect in any React application. The documentation file comes from the Storybook Docs addon [(docs)](https://github.com/storybookjs/storybook/tree/master/addons/docs) and works directly with your stories [(docs)](https://github.com/storybookjs/storybook/tree/master).
```

Comment:
I'm not sure you need this imo, the list of files is probably enough

Reply:
I wanted to explain addons somewhere. I'll just add a link to the list item